lib/screens/home_screen.dart

```diff
@@ -119,33 +122,266 @@ class _HomeScreenState extends State<HomeScreen> {
+                                                      children: <Widget>[
+                                                        Icon(
+                                                          Icons.star,
+                                                          color: Colors.red,
+                                                          size: media.orientation ==
+                                                                  Orientation
+                                                                      .portrait
+                                                              ? size.width *
+                                                                  0.045
+                                                              : size.width *
+                                                                  0.0283,
+                                                        ),
+                                                        Text(
+                                                          '1000',
+                                                          style: TextStyle(
+                                                              color:
+                                                                  Colors.white,
+                                                              fontSize: media
+                                                                          .orientation ==
+                                                                      Orientation
+                                                                          .portrait
+                                                                  ? size.width *
+                                                                      0.034
+                                                                  : size.width *
+                                                                      0.02),
+                                                        ),
+                                                      ],
+                                                    )
+                                                  ],
+                                                )
+                                              ],
+                                            )
+                                          ],
+                                        )),
+                                    Padding(
+                                      padding: EdgeInsets.fromLTRB(
+                                          0,
+                                          0,
+                                          media.orientation ==
+                                                  Orientation.portrait
+                                              ? size.width * 0.2
+                                              : size.width * 0.32,
+                                          0),
+                                      child: Text('Profile',
```

Comment:
try to create functions to render when you have similar items. Since there are 3 similar items, create a buildMenuItem or similar function and put the UI logic there